docs/data/data-grid/list-view/list-view.md

```diff
@@ -16,14 +16,106 @@ Unlike the default grid view, the list view makes no assumptions on how data is
+## Field visibility
+
+Field visibility can be toggled in list view, similarly to the default grid view.
+In the list view column's `renderCell` function, data can be conditionally rendered based on [column visibility](/x/react-data-grid/column-visibility/), as shown in the code snippet below.
+
+```tsx
+import {
+  useGridSelector,
+  useGridApiContext,
+  gridColumnVisibilityModelSelector,
+} from '@mui/x-data-grid';
+
+function ListViewCell(params: GridRenderCellParams) {
+  const apiRef = useGridApiContext();
+  const columnVisibilityModel = useGridSelector(
+    apiRef,
+    gridColumnVisibilityModelSelector,
+  );
+  const showCreatedAt = columnVisibilityModel.createdBy !== false;
+
+  return (
+    <>
+      <span>{params.row.id}</span>
+      {showCreatedAt && (
+        <time datetime={params.row.createdAt}>
+          {formatDate(params.row.createdAt)}
+        </time>
+      )}
+    </>
+  );
+}
+```
 
 ## Advanced usage
 
 The list view feature can be combined with [custom subcomponents](/x/react-data-grid/components/) to provide an improved user experience on small screens.
 
-{{"demo": "ListViewAdvanced.js", "bg": "inline", "iframe": true, "maxWidth": 360, "height": 600, "hideToolbar": true}}
+{{"demo": "ListViewAdvanced.js", "iframe": true, "maxWidth": 360, "height": 600}}
+
+## Feature support
+
+The list view feature can be used in combination with the following features:
+
+- ✅ [Sorting](/x/react-data-grid/sorting/)
+- ✅ [Filtering](/x/react-data-grid/filtering/)
+- ✅ [Multi filters](/x/react-data-grid/filtering/multi-filters/) [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+- ✅ [Row pinning](/x/react-data-grid/row-pinning/) [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+- ✅ [Pagination](/x/react-data-grid/pagination/)
+- ✅ [Row selection](/x/react-data-grid/row-selection/)
+- ✅ [Cell selection](/x/react-data-grid/cell-selection/) [<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan 'Premium plan')
+
+:::warning
+Features not listed may not work as expected, or may not work at all.
+
+If you're using a feature that's listed above and it's not working as expected, please [open a bug report](https://github.com/mui/mui-x/issues/new?assignees=&labels=status%3A+waiting+for+maintainer%2Cbug+%F0%9F%90%9B&projects=&template=1.bug.yml).
 
-:::info
-See the code for this demo in [CodeSandbox](https://codesandbox.io/p/sandbox/x-react-data-grid-list-view-zmkzhz).
+If you need to use list view with any other features, please [open a feature request](https://github.com/mui/mui-x/issues/new?assignees=&labels=status%3A+waiting+for+maintainer%2Cnew+feature&projects=&template=2.feature.yml).
 :::
+
+## Selectors
+
+{{"component": "modules/components/SelectorsDocs.js", "category": "List View"}}
+
+## API
+
+- [DataGrid](/x/api/data-grid/data-grid/)
+- [DataGridPro](/x/api/data-grid/data-grid-pro/)
+- [DataGridPremium](/x/api/data-grid/data-grid-premium/)
```

Comment:
Since we have the interface documented, does it make sense to include it here?
```suggestion
- [DataGridPremium](/x/api/data-grid/data-grid-premium/)
- [GridListColDef](/x/api/data-grid/grid-list-col-def/)
```